docs/nodes/light-node.mdx

```diff
@@ -165,26 +167,26 @@ so we must install Golang to build and run them.
 <Tabs groupId="operating-systems">
 <TabItem value="amd" label="Ubuntu (AMD)">
 
-```sh
-ver="1.19.1"
-cd $HOME
-wget "https://golang.org/dl/go$ver.linux-amd64.tar.gz"
-sudo rm -rf /usr/local/go
-sudo tar -C /usr/local -xzf "go$ver.linux-amd64.tar.gz"
-rm "go$ver.linux-amd64.tar.gz"
-```
+<pre><code>
+ver="{versions.golang}" <br/>
```

Comment:
this is an example of using the golang version as a variable. 

The `<br/>` tags are html breaks to make the code block multiline as the back tick blocks don't work for variables.

Reply:
I'll have to test out the difference in <pre><code> vs. <CodeBlock>, I think <CodeBlock> is not necessary, as it doesn't improve formatting in this case

Reply:
just tested it out, this rocks

Reply:
only thing that is missing here is the "copy" and "wrap" buttons 
<img width="163" alt="Screenshot 2023-02-03 at 8 05 11 PM" src="https://user-images.githubusercontent.com/46639943/216736938-a7072654-840f-4bca-8610-97c9d99b92ba.png">

Reply:
https://www.loom.com/share/5868d3252d154e1b8c05b216b66bfbd3 turns out they do work